src/map/clif.cpp

```diff
@@ -15832,7 +15832,7 @@ void clif_Mail_setattachment( map_session_data* sd, int index, int amount, uint8
 
 			p.weight += sd->mail.item[i].amount * ( sd->inventory_data[sd->mail.item[i].index]->weight / 10 );
 		}
-		p.favorite = item->favorite;
+		p.favorite = ( item->favorite != 0 ) ? 1 : 0;
```

Comment:
```suggestion
		p.favorite = ( item->favorite != 0 ) ? true : false;
```

Reply:
Same as above, favorite from `PACKET_ZC_ACK_ADD_ITEM_RODEX` is uint8